--- a/drivers/bluetooth/btrtl.c	2026-06-11 12:49:32.943948005 +0000
+++ b/drivers/bluetooth/btrtl.c	2026-06-11 12:49:32.947948005 +0000
@@ -188,6 +188,13 @@
 	  .cfg_name = "rtl_bt/rtl8723fu_config",
 	  .hw_info  = "rtl8733bu" },
 
+	/* 8733BS (UART, e.g. GameMT E5 Plus combo chip) */
+	{ IC_INFO(RTL_ROM_LMP_8723B, 0xf, 0x8, HCI_UART),
+	  .config_needed = true,
+	  .has_rom_version = true,
+	  .fw_name  = "rtl_bt/rtl8733bs_fw",
+	  .cfg_name = "rtl_bt/rtl8733bs_config",
+	  .hw_info  = "rtl8733bs" },
 
 	/* 8821A */
 	{ IC_INFO(RTL_ROM_LMP_8821A, 0xa, 0x6, HCI_USB),
@@ -1539,6 +1546,8 @@
 MODULE_FIRMWARE("rtl_bt/rtl8723d_fw.bin");
 MODULE_FIRMWARE("rtl_bt/rtl8723d_config.bin");
 MODULE_FIRMWARE("rtl_bt/rtl8723ds_fw.bin");
+MODULE_FIRMWARE("rtl_bt/rtl8733bs_fw.bin");
+MODULE_FIRMWARE("rtl_bt/rtl8733bs_config.bin");
 MODULE_FIRMWARE("rtl_bt/rtl8723ds_config.bin");
 MODULE_FIRMWARE("rtl_bt/rtl8723fu_fw.bin");
 MODULE_FIRMWARE("rtl_bt/rtl8723fu_config.bin");
